nidx/src/scheduler.rs

```diff
@@ -166,6 +168,24 @@ pub async fn run_tasks(
         }
     });
 
+    if let Some(audit_settings) = &settings.audit {
+        let audit_nats = async_nats::connect(&audit_settings.nats_server).await?;
+        let meta2 = meta.clone();
+
+        // Old date on startup to send all the reports, since we don't know how much time
+        // we have been offline.
+        let mut since = PrimitiveDateTime::MIN.replace_year(2000)?;
```

Comment:
this will report from the beginning of time every time it starts up no?

Reply:
ok, so at startup it will report all kbs, and since the values are absolute, we're ok with this?

Reply:
Yeah, I confirmed with Carles.